src/module/canvas/helpers.ts

```diff
@@ -18,14 +18,34 @@ function measureDistanceRect(
         return canvas.grid.measureDistance(r0, r1);
     }
 
+    const { dx, dy } = measureDistanceRectDimensions(r0, r1);
+
+    return measureDistanceOnGrid({ dx, dy }, { reach });
+}
+
+/**
+ * Measure the minimum distance between two rectangles on x and y Dimensions
+ * @param r0      The origin rectangle
+ * @param r1      The destination rectangle
+ */
+function measureDistanceRectDimensions(r0: PIXI.Rectangle, r1: PIXI.Rectangle): { dx: number; dy: number } {
```

Comment:
Given how little `measureDistanceRect` does now, does it even need to exist anymore?

Reply:
I will look into where it is used and if that can be migrated. Chances are Probably yes.

Reply:
refactored so that `measureDistanceRect` doesn't exist anymore. Tellme how wrong my method to achieve that was :)

also played around with gridless so it isn't pure dumb anymore.